x-pack/plugins/secops/public/components/flyout/index.test.tsx

```diff
@@ -204,7 +205,7 @@ describe('Flyout', () => {
     });
 
     test('should call the onOpen when the mouse is clicked for rendering', () => {
-      const showTimeline = jest.fn();
+      const showTimeline = (jest.fn() as unknown) as ActionCreator<{ id: string; show: boolean }>;
```

Comment:
note: with even stricter types using ActionCreator, `jest.fn()` needs me to use this ugly casting to work. If there's another way I would like to know.